src/safeds/ml/classical/classification/_classifier.py

```diff
@@ -136,3 +136,80 @@ def precision(self, validation_or_test_set: TaggedTable, positive_class: int = 1
         if (n_true_positives + n_false_positives) == 0:
             return 1.0
         return n_true_positives / (n_true_positives + n_false_positives)
+
+    def recall(self, validation_or_test_set: TaggedTable, positive_class: int = 1) -> float:
```

Comment:
```suggestion
    def recall(self, validation_or_test_set: TaggedTable, positive_class: Any = 1) -> float:
```

Need to import `typing.Any`.

Might also need to be changed for `precision`.